Pearson-Advance/edx-platform
# PADV 514 - BOM changes or features.

## Ticket

https://agile-jira.pearson.com/browse/PADV-514

## Description

This PR replaces the default email templates with a Pearson branded ones (due to an issue with themed email templates more context on PAE-342 located in JIRA backup) and adds the Pearson privacy policy extension point to the registration page.

## Type of Change

- [x] Cherry-pick of #37 and apply fixes to adjust style to Olive
- [x] Cherry-pick of #73 

## How to test

For registration feature:

- Install Pearson core
- Apply the following site configurations
```
{
    "REQUIRE_TOS_AND_PRIVACY_POLICY_AT_REGISTRATION": true,
    "REGISTRATION_EXTRA_FIELDS": {
        "honor_code": "hidden",
        "terms_of_service": "required"
    }
}
```
- You should now see the custom registration form in the registration page.

For the email templates

- Setup an async worker https://3.basecamp.com/3966315/buckets/12732851/messages/2842731569
- Test sending the corresponding emails and verify the styles are applied correctly

## Screenshots

https://imgur.com/a/tx1yzU6

## Reviewers
